lib/cc/analyzer/engines_config_builder.rb

```diff
@@ -11,9 +19,11 @@ def initialize(registry:, config:, container_label:, source_dir:, requested_path
         @source_dir = source_dir
       end
 
-      def run(engine_class = Analyzer::Engine)
+      def run
         names_and_raw_engine_configs.map do |name, raw_engine_config|
-          build_engine(engine_class, name, raw_engine_config)
+          label = @container_label || SecureRandom.uuid
```

Comment:
The desired behavior, based at least on the previous code, was to call `SecureRandom.uuid` once for each engine, so I'm just preserving that here.